lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/DrivingPermitHandler.java

```diff
@@ -76,38 +77,36 @@ public DrivingPermitHandler()
             throws NoSuchAlgorithmException, InvalidKeyException, CertificateException,
                     InvalidKeySpecException, HttpException, KeyStoreException, IOException {
         this.objectMapper = new ObjectMapper().registerModule(new JavaTimeModule());
-        ServiceFactory serviceFactory = new ServiceFactory(objectMapper);
-        this.eventProbe = new EventProbe();
-        this.identityVerificationService = serviceFactory.getIdentityVerificationService();
+        CommonServiceFactory commonServiceFactory = new CommonServiceFactory(objectMapper);
+        CRIServiceFactory criServiceFactory = new CRIServiceFactory(commonServiceFactory);
```

Comment:
@ElliotMurphyGDS I dont mind this idea but I think if were going down this road CRIServiceFactory can't take commonServiceFactory as a parameter otherwise we have coupled these two services together and I think if were creating things patterns to setup CRIs they should be capable of working independently